rero_ils/modules/users/validators.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2023 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Password validator."""
+
+from wtforms import ValidationError
+
+from rero_ils.modules.utils import PasswordValidatorException, \
+    password_validator
+
+
+class PasswordValidator:
+    """Password Validator."""
+
+    def __init__(self, length=8, special_char=False):
+        """Password validator class initializer."""
+        self.length = length
+        self.special_char = special_char
+
+    def __call__(self, form, field):
+        """Call.
+
+        :param form: the current form.
+        :param field: the password field.
+        :raise ValidationError: If the password isn't valid.
+        """
+        field_length = field.data and len(field.data) or 0
+        if field_length < self.length:
+            raise ValidationError(field.ngettext(
+                'Field must be at least %(min)d character long.',
+                'Field must be at least %(min)d characters long.',
+                self.length
+            ) % {"min": self.length})
```

Comment:
same remark. The length validation is already included into `password_validator`